test/JDBC/expected/PARTITION-vu-verify.out

```diff
@@ -1009,7 +1069,7 @@ varchar#!#int#!#int
 363863941f079adaa9aa733200e57c9f_partition_3_value_idx#!#4#!#3
 363863941f079adaa9aa733200e57c9f_partition_3#!#4#!#0
 363863941f079adaa9aa733200e57c9f_partition_3_id_key#!#4#!#2
-partition_vu_prepare_normal_inded6315af14d88f45711ba24e46851b8f#!#1#!#2
+partition_vu_prepare_normal_ind5f5fcd81a18b69e8a21aef69ce61f7b6#!#1#!#2
```

Comment:
1. How is partition_id getting changed with renaming index? Looks like an existing bug with sys.partitions. Please check with @sumitj824 
2. Also looking at this output, Eventually we should fix OBJECT_NAME to give the original name instead of hashed name.

Reply:
For 2nd, object_name/object_id functions only work for schema scoped functions but index is a table scoped object in T-SQL so these functions  are not actually meant to handle indexes.

Reply:
For 1st, the output is expected since sys.partitions returns entry for both tables and indexes and index `partition_vu_prepare_normal_index` has been removed to `partition_vu_prepare_normal_index_new` in the beginning using sp_rename so partition_id has also been changed.